src/setup/config.rs

```diff
@@ -2,24 +2,25 @@ use std::{
     collections::HashSet,
     ffi::OsString,
     fmt::Write,
-    fs, io,
-    net::{IpAddr, Ipv4Addr, SocketAddr},
+    fs,
+    net::{IpAddr, SocketAddr},
     path::PathBuf,
 };
 
 use anyhow::Result;
 use serde::Deserialize;
 
 // Ziggurat's configuration directory and file. Caches are written to this directory.
-const CONFIG: &str = ".ziggurat";
-const CONFIG_FILE: &str = "config.toml";
+pub const ZIGGURAT_CONFIG_DIR: &str = ".ziggurat";
```

Comment:
```suggestion
pub const ZIGGURAT_DIR: &str = ".ziggurat";
```
I now noticed that cache is put there as well

Reply:
Done